sim/hunter/raptor_strike.go

```diff
@@ -85,13 +89,17 @@ func (hunter *Hunter) getRaptorStrikeConfig(rank int) core.SpellConfig {
 				spell.BonusWeaponDamage()
 
 			if hasFlankingStrike && hunter.FlankingStrikeAura.IsActive() {
-				mhBaseDamage *= 1.0 + (0.1 * float64(hunter.FlankingStrikeAura.GetStacks()))
+				mhBaseDamage *= 1.0 + (flankingStrikeDmgMult * float64(hunter.FlankingStrikeAura.GetStacks()))
```

Comment:
set the flanking and raptor fury stacks into variables if you're using them in multiple lines. No point in calling `float64(GetStacks())` multiple times

Reply:
This is totally not needed for this PR but I noticed most of the conditions are checked for both MH and OH e.g. `hasFlankingStrike && hunter.FlankingStrikeAura.IsActive()` we should maybe do the same with those

Reply:
I'll look at cleaning this stuff up with this PR since I'm going to be making one more commit with minor changes anyway